sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -323,6 +326,59 @@ export class BatchingReceiverLite {
     }
   }
 
+  private async drainReceiverIfNeeded(
```

Comment:
nitpick: `tryDrainLink`?

Reply:
I still want to keep receiver in the name as we are waiting for receiverDrained event. Renaming to `tryDrainReceiver`